src/components/conversations/ConversationView.tsx

```diff
@@ -216,27 +216,18 @@ const ConversationView: React.FC<SingleConvoProps> = (
               textOverflow: "ellipsis",
             }}
           >
-            {!props.typingInfo.length ? (
-              <Box>
+            {!props.typingInfo.length && lastMsgStatus ? (
+              <Box display="flex" paddingRight={"space20"} alignItems="center">
                 {lastMsgStatus === MessageStatus.Sending && props.myMessage && (
-                  <Box style={{ paddingRight: 6 }}>
```

Comment:
no need for an extra box around each icon, and makes it harder to vertically center